mypy/checkexpr.py

```diff
@@ -1,42 +1,37 @@
 """Expression type checker. This file is conceptually part of TypeChecker."""
 
-from typing import cast, List, Tuple, Dict, Callable, Union, Optional
+from typing import cast, List, Tuple, Callable, Union, Optional
 
-from mypy.types import (
-    Type, AnyType, CallableType, Overloaded, NoneTyp, Void, TypeVarDef,
-    TupleType, Instance, TypeVarType, TypeTranslator, ErasedType, FunctionLike, UnionType,
-    PartialType, DeletedType
-)
+import mypy.checker
+from mypy import applytype
+from mypy import erasetype
+from mypy import join
+from mypy import messages
+from mypy import nodes
+from mypy import types
```

Comment:
My opinion would be yes, but if Guido doesn't want to move around the imports, that's a moot point.